### PR TITLE
Fix Integration Tests

### PR DESCRIPTION
**Pull request by BlairArchibald**
Fix integration tests.

TestLockdown is still failing/ignored as I am unsure of the final check and this will need to epidemiological input.

### CODE REVIEW

File: src/test/java/uk/co/ramp/covid/simulation/ModelTest.java [hunk 1]
```diff
@@ -183,6 +181,8 @@ public void testLockdown() {
             }
         }
         assertTrue("Unexpectedly fewer infections before lockdown", totInfDuringLockdown < totInfBeforeLockdown);
+        // TODO-CHECK: I'm unsure this is a correct test. If the lockdown is very effective there
+        //  are very low cases in the population afterwards
```
Comment (bobtaylor12) [inline src/test/java/uk/co/ramp/covid/simulation/ModelTest.java:185]: The test fails for me on the assert at line 167. We should expect the total number of infections to be lower in the lockdown scenario.

Comment (bobtaylor12) [reply src/test/java/uk/co/ramp/covid/simulation/ModelTest.java:185]: You're right. The assert at line 186 probably doesn't make sense

File: src/test/java/uk/co/ramp/covid/simulation/population/ShiftsTest.java [hunk 2]
```diff
@@ -14,13 +14,14 @@
...
 
-    @Ignore("Failing Test")
     @Test
     public void testGetShift() throws JsonParseException, IOException {
         ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
 
        Shop shop = new Shop(CommunalPlace.Size.LARGE);
        int end = shop.getShifts().getShift(1).getEnd();
+       assertEquals(15, end);
+       end = shop.getShifts().getShift(1).getEnd();
```
Comment (bobtaylor12) [inline src/test/java/uk/co/ramp/covid/simulation/population/ShiftsTest.java:24]: Not sure I understand how this works but both asserts pass now

File: src/test/java/uk/co/ramp/covid/simulation/ModelTest.java (hunk 1 above)
Comment (BlairArchibald) [reply src/test/java/uk/co/ramp/covid/simulation/ModelTest.java:185]: Does it still fail for you on 167? I couldn't reproduce that (could be fixed when I fixed the movement).

File: src/test/java/uk/co/ramp/covid/simulation/population/ShiftsTest.java (hunk 2 above)
Comment (BlairArchibald) [reply src/test/java/uk/co/ramp/covid/simulation/population/ShiftsTest.java:24]: Shifts is an infinitely long queue of possible shift patterns (for when we assign workers) for a given day. For shops the first shift you get is 8-15, then 15-22, 8-15, 15-22, and so on.

**Comment by bobtaylor12** [comment]
Thanks for that. Yes the test at line 167 still fails but I probably don't
have the latest code yet.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by johnnonweiler** [comment]
Please remove any imports which are no longer required (such as `import org.junit.Ignore;`)

**Comment by johnnonweiler** [comment]
I've now discovered that our gradle build won't tell you about unused imports, so, unless you're using an IDE which does, you probably shouldn't worry about it.

**Comment by johnnonweiler** [comment]
What is this PR waiting for?  (It looks like it's either waiting for changes following Bob's review, or approval by Bob, but I'm not sure which.)

**Comment by johnnonweiler** [comment]
The unused imports will be removed in #59 . (I accidentally removed them at the same time as doing a rebase to fix merge conflicts.)

**Pull request merged**